roles/cluster_compare/tasks/upstream.yml

```diff
@@ -0,0 +1,102 @@
+---
+- name: Get the latest tools release tag
+  ansible.builtin.uri:
+    url: "https://api.github.com/repos/openshift/kube-compare/releases/latest"
+    method: GET
+    return_content: true
+    status_code: 200
+  register: _cc_github_response
+  no_log: "{{ cc_logging }}"
+
+- name: Download and extract binary files
```

Comment:
It can be avoided to get the latest tag by directly downloading the latest release, e.g.
- https://github.com/openshift/kube-compare/releases/latest/download/kube-compare_linux_amd64.tar.gz
- https://github.com/openshift/kube-compare/releases/latest/download/kube-compare_addon_tools_linux_amd64.tar.gz